index.js

```diff
@@ -377,7 +377,7 @@ async function apostrophe(options, telemetry, rootSpan) {
     const reallyLocalPath = self.rootDir + localPath;
 
     if (fs.existsSync(reallyLocalPath)) {
-      local = await self.root.import(reallyLocalPath);
+      local = await self.root.import(pathToFileURL(reallyLocalPath));
```

Comment:
"import" takes URLs, not filenames, although file:// is the default protocol. So without this, "C:" is treated as some newly invented protocol, and you are busted.